### ISSUE TITLE
backend crash when using v4.1 appimage to generate a moment image

### ISSUE THREAD

**Issue by kswang1029** [CARTAvis/carta#206]
This is a user input via the Helpdesk. If we use the v4.1 appimage (tested on RH8 or Ubuntu22.04) to generate a moment image from a FITS cube (this one is with per-plane-beam), the backend will crash. However, with the v4.1 rpm, a moment image can be successfully generated. This implies it is a packaging issue.

please contact @kswang1029 to get a copy of the test image

**Comment by kswang1029** [comment]
UPDATE: I confirm that it is due to the image contains a per-plane-beam table. If we convolve the cube with a common beam size, then with the v4.1 AppImage, we can generate a moment image successfully.

**Comment by lboogaard** [comment]
I can confirm this issue for cubes with multiple beam planes in version 4.1.0. 
Will the above fix be part of the next release (when can this be expected)?

**Comment by kswang1029** [comment]
> I can confirm this issue for cubes with multiple beam planes in version 4.1.0. Will the above fix be part of the next release (when can this be expected)?

Yes, the packaging issue should have been fixed by updating our build and packaging environment. The upcoming v5-beta and v5-stable release should not have the issue.
